OneSignalSDK/onesignal/build.gradle

```diff
@@ -5,7 +5,7 @@ ext {
 
     androidXVersion = '[1.0.0, 1.99.99]'
     androidWorkVersion = '[2.0.0, 2.99.99]'
-    firebaseMessagingVersion = '[19.0.0, 20.99.99]'
+    firebaseMessagingVersion = '[19.0.0, 21.99.99]'
```

Comment:
Can you put more information about the error and what specific under this version solve the problem?

Reply:
Yes, there is a known issue https://firebase.google.com/support/release-notes/android#messaging_v20-2-2 and we are asked not to use this version. For some reason I couldn't find when the version range is set to [19.0.0, 20.99.99] on my builds didn't go for 20.3.0 which already has a fix for that.

Reply:
@jkasten2 FYI